geoconvert/data/countries.py

```diff
@@ -48,6 +48,12 @@
     "haute vienne": "FR",  # fr
 }
 
+# We want to make sure pacific island is not registered as IS (Iceland) in German.
+# Other languages doesn't have issues with it and we shouldn't prevent them from retrieving
+# other countries like FJ (Fiji) in "Fiji/Pacific Island"
+special_countries_de = {
+    "pacific island": None,  # en
+}
```

Comment:
Actually we want to make sure that the word _island_ never matches Iceland in an english text (not just Pacific Island)